nixos/modules/services/web-apps/moin.nix

```diff
@@ -0,0 +1,291 @@
+{ config, lib, pkgs, ... }:
+with lib;
+
+let
+  cfg = config.services.moin;
+  python = pkgs.python27;
+  pkg = python.pkgs.moinmoin;
+  dataDir = "/var/lib/moin";
+  usingGunicorn = cfg.webServer == "nginx-gunicorn" || cfg.webServer == "gunicorn";
+  usingNginx = cfg.webServer == "nginx-gunicorn";
+  user = "moin";
+  group = "moin";
+
+  uLit = s: ''u"${s}"'';
+  indentLines = n: str: concatMapStrings (line: "${fixedWidthString n " " " "}${line}\n") (splitString "\n" str);
+
+  moinCliWrapper = wikiIdent: pkgs.writeShellScriptBin "moin-${wikiIdent}" ''
+    ${pkgs.su}/bin/su -s ${pkgs.runtimeShell} -c "${pkg}/bin/moin --config-dir=/var/lib/moin/${wikiIdent}/config $*" ${user}
```

Comment:
If you didn't use `su` but instead just ran the command then nixos sysadmins could have better control over the access to this script with `sudo` rules. The sysadmin can always type `sudo -u moin-ExampleWiki` to be explicit.

Reply:
I wanted to make setting up simple instance as easy as possible, not sure whether a script that just adds single argument to the `moin` cli helps with that. For maximum flexibility the `moin` cli is added to `systemPackages` as well.

Or did you mean to use sudo with some default rules instead?

Reply:
Personally I don't like the approach taken here (a script using `su`) and my preference would certainly be to document setting up access via `sudo` rules. I want to be clear on this point what I state is only my **opinion**, and not a hard requirement preventing this PR from being merged. You may take either approach.